packages/@aws-cdk/aws-cloudwatch/lib/dashboard.ts

```diff
@@ -67,6 +69,16 @@ export class Dashboard extends Resource {
       physicalName: props.dashboardName,
     });
 
+    {
+      const {dashboardName} = props;
```

Comment:
what's wrong with `props.dashboardName`?

Reply:
It's just a personal preference, I destructure variables instead of accessing them multiple times. I can revert it if you want